20-packages.ks

```diff
@@ -11,29 +16,23 @@ chkconfig
 rootfiles
 yum
 vim-minimal
-acpid
-tftp
 lldpad
-isomd5sum
```

Comment:
I think this way needed so ISO checksum can be called from the bootloader (check iso and boot). Not sure about it, but I think it goes into initramdisk when it is built, then it is not needed after boot of course.